### ISSUE TITLE
[Settings]: The 'Open JSON File' element's ControlType requires valid values for SizeOfSet and PositionInSet.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11154]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:** 
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1689.01

### Steps to reproduce

**Repro Steps:**

1. Open Windows Terminal.
2. Open Settings page using "Ctrl + ',' ".
3. Run Accessibility insights for Window on Settings page and Observe the issue.

**User Experience:** -> _Priority 2_
Screen Reader user are impacted here as incorrect position is announced for control which will confuse user.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[The 'Open JSON File' element's ControlType requires valid values for SizeOfSet and PositionInSet..zip](https://github.com/microsoft/terminal/files/7115533/The.Open.JSON.File.element.s.ControlType.requires.valid.values.for.SizeOfSet.and.PositionInSet.zip)

### Expected Behavior

The 'Open JSON File' element's ControlType requires valid values for SizeOfSet and PositionInSet.

### Actual Behavior

The 'Open JSON File' element's ControlType doesn't have valid values for SizeOfSet and PositionInSet. Screen Reader announce 'Open JSON file 11 of 11' which is incorrect.

**Comment by v-rpundir** [comment]
**Bug Update in Dec Test Pass:** Issue Still repro as Screen Reader is only announcing name as 'Open JSON file' when user navigate to 'Open JSON file' control using down arrow key.

**Comment by carlos-zamora** [comment]
I think this might be an external issue, if it's still even considered an issue at all?

Here's some background...
### `SizeOfSet` and `PositionInSet` have now changed
> The 'Open JSON File' element's ControlType doesn't have valid values for SizeOfSet and PositionInSet. Screen Reader announce 'Open JSON file 11 of 11' which is incorrect.

Not sure when this was introduced, but on 15a047512, I'm not seeing `SizeOfSet` and `PositionInSet` both being `11`. Instead, Accessibility Insights is telling me `Property does not exist`.

### What do other apps do
#### Edge - vertical tabs
Interesting! The "New tab" button there experiences the same behavior.

#### XAML Controls Gallery
The "Settings" item (which is located in the footer) is marked `SizeOfSet: 1` and `PositionInSet: 1`. 

### What is the correct behavior here?
In Windows Terminal's settings UI, I'm seeing the navigation view items organized into three sets:
- Set 1 includes "Startup" through "Actions"
- Set 2 includes "Defaults", all of the profiles, and "Add a new profile"
- Set 3 includes "Open JSON file"

@v-rpundir What's the correct behavior here?
1. [Current behavior] keep `SizeOfSet` and `PositionInSet` as undefined. Note that this matches Edge's vertical tabs.
2. [Match XAML Controls Gallery] `SizeOfSet` and `PositionInSet` should both be `1`
3. [Everything is wrong?] The three sets above should all be one contiguous set

CC @DHowett @zadjii-msft for thoughts

**Comment by v-rpundir** [comment]
Hi @carlos-zamora 

This is correct behavior as it is universal throughout Windows Apps.
**Case 2: [Match XAML Controls Gallery] SizeOfSet and PositionInSet should both be 1**

Below one is current situation, but Screen Reader is still announcing incorrect position & AI is giving Error:
For Case 1: [Current behavior] keep SizeOfSet and PositionInSet as undefined. Note that this matches Edge's vertical tabs.
<img width="594" alt="image" src="https://user-images.githubusercontent.com/86412703/151294165-a1aaef76-7e91-4427-b42a-702bfc6a7fc5.png">


Thanks :):

**Comment by carlos-zamora** [comment]
@v-rpundir Thanks!

I tried setting fixing `SizeOfSet` and `PositionInSet` manually. That's not working.

I've reached out to the XAML team [here](https://github.com/microsoft/microsoft-ui-xaml/issues/1971#issuecomment-1024621245), as this is an upstream issue that we are blocked by. We're going to have to tag this one as a "tracking external" bug.

**Comment by carlos-zamora** [comment]
Nevermind! Worked with the XAML team and found something that works. This'll be fixed by #12286.

**Comment by v-rpundir** [comment]
Closing as fixed in Windows Terminal Preview Version: 1.13.10336.0
